Source/WebCore/platform/cocoa/PlaybackSessionModel.h

```diff
@@ -72,7 +72,7 @@ class PlaybackSessionModel : public CanMakeWeakPtr<PlaybackSessionModel> {
     virtual void selectAudioMediaOption(uint64_t index) = 0;
     virtual void selectLegibleMediaOption(uint64_t index) = 0;
     virtual void togglePictureInPicture() = 0;
-    virtual void toggleInWindow() = 0;
+    virtual void toggleInWindowFullscreen() = 0;
```

Comment:
The `In` in the name doesn’t seem necessary. How about `toggleWindowFullscreen`?

Reply:
The "in" is "in"portant. :) 

This is about entering an in-window fullscreen, so dropping the "in" would change the meaning.